workflow/catalog.py

```diff
@@ -1,47 +1,38 @@
 import pandas as pd
-import numpy as np
-import glob
 import os
 from pathlib import Path
-import argparse
-import json
-
-# Globals
-# mr_proc_manifest columns
-PARTICIPANT_ID = "participant_id"
-PARTICIPANT_DICOM_DIR = "participant_dicom_dir"
-VISIT = "visit"
-SESSION = "session"
-DATATYPE = "datatype"
-DICOM_ID = "dicom_id"
-BIDS_ID  = "bids_id"
-
-def read_manifest(manifest_csv, session_id, logger):
+
+from workflow.utils import (
+    COL_BIDS_ID_MANIFEST,
+    COL_CONV_STATUS,
+    COL_DICOM_DIR,
```

Comment:
I had the `PARTICIPANT_DICOM_DIR` to avoid confusion with the parent `dicom` dir in the mr_proc dir_tree. Is there a reason you prefer otherwise?

Reply:
No particular reason, I think I was just trying to shorten the variable name but since that can be confusing with the `dicom` directory then I'll change `COL_DICOM_DIR` to `COL_PARTICIPANT_DICOM_DIR`